### ISSUE TITLE
Bug: Run eslint --quiet shows warnings but not errors

### ISSUE THREAD

**Issue by alper** [eslint/eslint#18612]
### Environment

Node version: 
npm version: 
Local ESLint version: 
Global ESLint version: 
Operating System: 

### What parser are you using?

Default (Espree)

### What did you do?

<details>
<summary>Configuration</summary>

```
<!-- Paste your configuration here -->
```
</details>

```js
<!-- Paste your code here -->
```

### What did you expect to happen?

It should show errors.

### What actually happened?

It fails but shows only warnings.

### Link to Minimal Reproducible Example

no

### Participation

- [X] I am willing to submit a pull request for this issue.

**Comment by mdjermanovic** [comment]
Please fill out the issue template and provide a reproducible example so we could take a look. I've just tested `--quiet` locally and it correctly shows errors.

**Comment by alper** [comment]
This is very weird. Feel free to close it. Our setup is relatively complex and eslint is being run by a custom nx executor and nowhere do we pass `--quiet`.

I can't make a small version of this but I have it down to investigate at some point in the future.

**Comment by mdjermanovic** [comment]
Okay, closing as there's not enough information to proceed.
